### ISSUE TITLE
$_FILES array contains trailing brackets, does not handle array field name

### ISSUE THREAD

**Issue by gordon-matt** [peachpiecompiler/peachpie#696]
I am attempting to get the latest version of Responsive File Manager working with PeachPie. There is a `NullReferenceException` thrown when uploading. Please refer to the screenshot below:

![NullRefException](https://user-images.githubusercontent.com/2860937/76143482-b5054000-60c7-11ea-94dd-38e3dea32e34.PNG)

In previous versions, it was a single file upload. Now they have changed it to support multiple file upload. As such, the code highlighted above used to be:

`$mime_type = mime_content_type($_FILES['file']['tmp_name']);`

and is now:

`$mime_type = mime_content_type($_FILES['files']['tmp_name'][0]);`

So they basically changed it to an array. Now notice that the name of the array is `files` and they are simply referring to it that way. However, PeachPie is using the name `files[]` (see the brackets added).

I believe this is why I am seeing the NullReferenceException.

It is possible I will see this in multiple other places in the code as well.

Is this a bug in PeachPie? What can I do?

**NOTE**: You can see the bug for yourself in [my code](https://github.com/gordon-matt/peachpie-responsive-file-manager/tree/feature/RFM9_14_0). Ensure you are looking at the feature branch named, **RFM9_14_0**

**Comment by jakubmisek** [comment]
Thank you @gordon-matt ! It seems like a bug in PeachPie. The entry should be named `"files"` without brackets.

**Comment by jakubmisek** [comment]
to try the latest version, you can change the peachpie version to the prview one:
- in `global.json` change version to `1.0.0-appv3541` or newer
- add `NuGet.config` file into the project root:
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="appveyor" value="https://ci.appveyor.com/nuget/peachpie" />
  </packageSources>
</configuration>
```

**Comment by gordon-matt** [comment]
Excellent! That works well now. When can we expect the next release?

**Comment by jakubmisek** [comment]
Good it works. Please let's test it for a while, maybe there'll be more issues to fix before we make a public release. Also you can occasionally try to update the preview version to the latest one if it's still working (or maybe setup some automatic tests?).

We can expect the new release in a week or so.

**Comment by jakubmisek** [comment]
@gordon-matt just released version `0.9.910` which includes the fix
